### ISSUE TITLE
Use system keyboard layout explanation missing

### ISSUE THREAD

**Issue by baimafeima** [ibus/ibus#1906]
When accessing _Preferences_ with a right-click, there is an option to "use system keyboard layout" under the _Advanced Tab_. The tooltip merely adds "XKB" to it without adding any help in explaining what this option is about. It would be helpful to add a few lines directly below this option (instead of the tooltip) to explain what the effects of this option are and for whom they are advisable.

**Comment by fujiwarat** [comment]
IBus runs `setxkbmap` command internally to switch ibus engines and set the keymaps.
The option means, if the option is disabled, ibus does not change the system keyboard layout with `setxkbmap`.
I don't think it's a good way to add another label to explain the button.
Probably updating the man page; ibus-setup(1) makes sense.
It's welcome to contribute to update the man page:
https://github.com/ibus/ibus/blob/master/setup/ibus-setup.1.in

**Comment by baimafeima** [comment]
Unfortunately, I don't understand your explanation. I just point out issues which I find confusing and many new users migrating to Linux-based operating systems might find equally confusing. The general rule I follow is: if it is not immediately understandable, there should be a way to get that information via a help button.

**Comment by fujiwarat** [comment]
> there should be a way to get that information via a help button.

Disagreed. We don't have such a way.

**Comment by hugobuddel** [comment]
It is totally unclear to me either. IBus does nothing when "use system keyboard layout" is checked on my machine (Ubuntu 16.04.2 with MATE). The keyboard layout is set to, well, the system keyboard layout (as the option implies), and I cannot change this through IBus until I unchecked the box.

"Use system keyboard layout" is checked by default. What is the point of having a keyboard layout switcher tool if the default is to 'use the system keyboard layout' anyway irrespective of which layout is selected?

**Comment by fujiwarat** [comment]
> "Use system keyboard layout" is checked by default

No, it's not a default setting. You can check the button with a new user account.

**Comment by fujiwarat** [comment]
> Perhaps it is only the default in Ubuntu 16.04 or Ubuntu MATE

I don't think so but your personal setting.

**Comment by hugobuddel** [comment]
See attached screenshot. Steps to reproduce:

1. Install Ubuntu 16.04.2 (allow it to update while installing).
2. Install `ubuntu-mate-desktop`.
3. Logout of unity, login with MATE.
4. Right click on IBus icon in tray, go to preferences, go to Advanced tab.
5. Make screenshot.

No personal settings were applied, just the defaults as they are out of the box.

Creating a new user shows the checkbox as well. This might be just an Ubuntu thing 16.04.2 thing. It is a long term release of Ubuntu, so it might be important to you that your software works as intended.

![defaults2](https://cloud.githubusercontent.com/assets/1306453/25804857/a186e834-33fd-11e7-8b6e-dd66d6061c57.png)

**Comment by fujiwarat** [comment]
OK, interesting. Let's talk with issue 1923.
